store/project_webhook.go

```diff
@@ -10,33 +10,163 @@ import (
 
 	"github.com/bytebase/bytebase/api"
 	"github.com/bytebase/bytebase/common"
-	"go.uber.org/zap"
 )
 
-var (
-	_ api.ProjectWebhookService = (*ProjectWebhookService)(nil)
-)
+// projectWebhookRaw is the store model for an ProjectWebhook.
+// Fields have exactly the same meanings as ProjectWebhook.
+type projectWebhookRaw struct {
+	ID int
+
+	// Standard fields
+	CreatorID int
+	CreatedTs int64
+	UpdaterID int
+	UpdatedTs int64
+
+	// Related fields
+	ProjectID int
+
+	// Domain specific fields
+	Type         string
+	Name         string
+	URL          string
+	ActivityList []string
+}
+
+// toProjectWebhook creates an instance of ProjectWebhook based on the projectWebhookRaw.
+// This is intended to be called when we need to compose an ProjectWebhook relationship.
+func (raw *projectWebhookRaw) toProjectWebhook() *api.ProjectWebhook {
+	projectWebhook := api.ProjectWebhook{
+		ID: raw.ID,
+
+		// Standard fields
+		CreatorID: raw.CreatorID,
+		CreatedTs: raw.CreatedTs,
+		UpdaterID: raw.UpdaterID,
+		UpdatedTs: raw.UpdatedTs,
+
+		// Related fields
+		ProjectID: raw.ProjectID,
+
+		// Domain specific fields
+		Type: raw.Type,
+		Name: raw.Name,
+		URL:  raw.URL,
+	}
+	projectWebhook.ActivityList = append(projectWebhook.ActivityList, raw.ActivityList...)
```

Comment:
nit: we don't need to copy the slice of []string?

Reply:
IMO if do not copy, the composed `ProjectWebhook` will have an empty `ActivityList`.